tests/test_cmc_client.py

```
@@ -0,0 +1,16 @@
+import os
+import sys
+from unittest.mock import patch
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+from src.data.cmc.cmc_client import CoinMarketCapClient
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)